pkg/operation/botanist/etcdencryption.go

```diff
@@ -0,0 +1,332 @@
+package botanist
+
+import (
+	"fmt"
+
+	gardenv1beta1 "github.com/gardener/gardener/pkg/apis/garden/v1beta1"
+	"github.com/gardener/gardener/pkg/operation/common"
+	encryptionconfiguration "github.com/gardener/gardener/pkg/operation/etcdencryption"
+	"github.com/gardener/gardener/pkg/utils"
+
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	apiserverconfigv1 "k8s.io/apiserver/pkg/apis/config/v1"
+	"k8s.io/client-go/util/retry"
+)
+
+// CreateEtcdEncryptionConfiguration creates a secret
+func (b *Botanist) CreateEtcdEncryptionConfiguration() error {
+	b.Logger.Info("starting CreateEtcdEncryptionConfiguration")
```

Comment:
Log message not needed - the flow which executes this function can log this if we want to have it

Reply:
ok, removed